src/017-auto-execution-of-command-after-connection.md

```diff
@@ -0,0 +1,38 @@
+# Automatically execute SQL after connecting to database
+
+SeaORM does not provide direct support for this matter, but is it possible to. The method involves establishing the connection with [`sqlx`](https://crates.io/crates/sqlx) then converting it back to a `sea_orm::DatabaseConnection` object.
+
+```rust, no_run
+// Only copy the one that you are using
+type DB = sqlx::Postgres;
+type DB = sqlx::Sqlite;
+type DB = sqlx::MySql;
+```
```

Comment:
What a smart way to avoid clutter while still able to mention all three supported databases!